demo/lightning/text-based/distributed/example-deploy.yaml

```diff
@@ -0,0 +1,72 @@
+# Use Static provisioning to define a PV for Cloud Storage buckets.
+# https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/cloud-storage-fuse-csi-driver#provision-static.
+apiVersion: jobset.x-k8s.io/v1alpha2
```

Comment:
Note that the YAML file here assumes that you have jobset and Kueue enabled so it will not work with any regular GKE cluster. I see that you have a README PR following up. We might as well ask that users create a cluster using XPK which comes with those things. In fact you are able to run this on our clusters because those are created by XPK.